src/timelines/preamble.js

```diff
@@ -1,46 +1,27 @@
-import { jsPsych } from "jspsych-react";
-import { eventCodes, lang } from "../config/main";
 import holdUpMarker from "../trials/holdUpMarker";
-import { showMessage, userId } from "@brown-ccv/behavioral-task-trials";
-import { config, AT_HOME } from "../config/main";
-console.log("at_home", AT_HOME);
-console.log("env at home", process.env.REACT_APP_AT_HOME);
+import startCode from "../trials/startCode";
+import welcome from "../trials/welcome";
+import adjustVolume from "../trials/adjustVolume";
+import camera from "../trials/camera";
+import { USE_EVENT_MARKER, VOLUME, VIDEO } from "../config/main";
+
+let timeline = [];
+if (VOLUME) {
+  timeline.push(adjustVolume());
+}
```

Comment:
this timeline doesn't look like what's in the old part of the diff

Reply:
@mcmcgrath13 About the config object, should I just use the variables from the object when they are needed, like instead of using USE_PHOTODIODE directly, I import the config object and use config.USE_PHOTODIODE instead? Also what variables should I add? Maybe USE_CAMERA?

Reply:
yup, definitely USE_CAMERA.  Any others that are used in trials, but in the feature category (env vars).

Reply:
@mcmcgrath13 Thanks. Also, relating back to the original comment, what do we want the preamble to look like. Right now I just followed what was in MSIT, is there a specific order we want the screens?

Reply:
Just recreate the preamble that was already in honeycomb